common/changes/separate-button-sass-fixes_2017-01-25-16-36.json

```diff
@@ -0,0 +1,15 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Restored Hero variant and added normal button as an alias of default",
```

Comment:
"Button: Restored Hero variant and added normal button as an alias of default. Fixes issues introduced in #811."